VCCrypto/VCCrypto/CryptoOperations.swift

```diff
@@ -3,78 +3,56 @@
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/
 
-public protocol CryptoOperating {
-    func generateKey() throws -> VCCryptoSecret
-    func retrieveKeyFromStorage(withId id: UUID) -> VCCryptoSecret
-    func save(key: Data, withId id: UUID) throws
-    func deleteKey(withId id: UUID) throws
-    func getKey(withId id: UUID) throws -> Data
+enum CryptoOperationsError: Error {
+    case invalidPublicKey
 }
 
+/// Operations that are involved in verification cryptographic operations..
 public struct CryptoOperations: CryptoOperating {
-
-    private let secretStore: SecretStoring
     
-    private let sdkConfiguration: VCSDKConfigurable
+    public init() {}
```

Comment:
Do we need unit tests for this class? And the Key management one?

Reply:
tests added.